packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -163,12 +168,15 @@ abstract class FlutterCommand extends Command<Null> {
     final List<bool> modeFlags = <bool>[argResults['debug'], argResults['profile'], argResults['release']];
     if (modeFlags.where((bool flag) => flag).length > 1)
       throw new UsageException('Only one of --debug, --profile, or --release can be specified.', null);
+    final bool dynamicFlag = argParser.options.containsKey('dynamic')
+        ? argResults['dynamic']
+        : false;
```

Comment:
Maybe just:
```
final bool dynamicFlag = argResults['dynamic'] ?? false;
```
since Map returns null when a key isn't present, and either way we'd never want `dynamicFlag` to be null.

Reply:
Unfortunately, this isn't a regular map. It overrides [] and will throw if flag doesn't exist. So this is the patterns that's used everywhere for argResults.